featuretools/feature_base/feature_visualizer.py

```diff
@@ -26,6 +26,16 @@
 
 
 def graph_feature(feat, to_file=None):
```

Comment:
nitpick, but let's avoid shortening variable names unless it's really needed. i think you can call this input `feature`